doc/devel/documenting_mpl.rst

```diff
@@ -121,8 +127,12 @@ You can use the ``O`` variable to set additional options:
 Multiple options can be combined using e.g. ``make O='-j4 -Dplot_gallery=0'
 html``.
 
-On Windows, options needs to be set as environment variables, e.g. ``set O=-W
---keep-going -j4 & make html``.
+On Windows, either use the format shown above or set options as environment variables, e.g.:
+
+.. code-block:: bat
+
+   set O=-W --keep-going -j4 
```

Comment:
As far as I understand SPHINXOPTS and O are identical (https://www.sphinx-doc.org/en/master/man/sphinx-build.html?highlight=makefile#environment-variables). It would be less confusing to consistently use only one of these throught the description.

That said, I don't know how they interact with each other (particularly also because `make.bat`  defines SPHINXOPTS).

Reply:
They're the same, except that `O` comes after `SPHINXOPTS`; mostly this is relevant if you want to override some part of the defaults. That being said, Sphinx doesn't have an option to turn off warnings, so using `O` for that doesn't work once `SPHINXOPTS` has `-W` in it.

And as an additional caveat, the `make.bat` doesn't work like a `Makefile`; I think it sets the environment variables unilaterally, and setting them before running it does nothing. So this section should be removed?